scripts/soroban-settings/SorobanSettingsUpgrade.py

```diff
@@ -0,0 +1,255 @@
+from stellar_sdk import xdr as stellar_xdr
+from stellar_sdk import Network, Keypair, TransactionBuilder, StrKey, utils
+from stellar_sdk.soroban import SorobanServer
+from stellar_sdk.soroban.types import Address, Int128, Bytes
+from stellar_sdk.soroban.soroban_rpc import GetTransactionStatus
+from stellar_sdk.xdr import TransactionMeta, LedgerKey, ConfigUpgradeSet, ConfigSettingContractExecutionLanesV0, ConfigUpgradeSetKey, ConfigSettingEntry, StateExpirationSettings, Uint32, Uint64, Int64, Hash, LedgerKeyConfigSetting, ConfigSettingID
```

Comment:
How about adding ConfigSettingContractLedgerCostV0 since that has read/write bytes and ledger settings there? We can throttle those using this script.

Reply:
Or we can add this in next iteration of the script.

Reply:
Ah! I see what the id argument corresponds to ConfigSettingEntry XDR union. So passing 9 would fetch us contract_data_entry_size_bytes.